### PR TITLE
making corrections to the code

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 1]
```diff
@@ -33,10 +43,22 @@ class BeerOfTheDayComponent extends PureComponent {
           onClick={this.getRandomBeer}
           text="Choose a beer"
         />
-        {beer && <RandomBeer randomBeer={beer} />}
+        {
+          name && <div key={id} className="random-beer">
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx:47]: key is used for map purpose - no need to add it here

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 2]
```diff
@@ -104,8 +104,8 @@ class SearchBeersComponent extends PureComponent {
   };
 
   getBeers = async () => {
-    const { alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food } = this.state;
-    const beers = await fetchBeers([alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food])
+    const { alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food, } = this.state;
+    const beers = await fetchBeers([alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food,])
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx:108]: No need to add trailing comas

File: src/services/beers.js [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
 };
 
 export async function fetchBeers(param) {
-  const url = `${baseUrl}/beers?${param.join('&')}`;
+  const url = `${baseUrl}/beers?${param.filter(String).join('&')}`
   try {
     const response = await fetch(url);
-    return (await response.json())
+    const beers = (await response.json())
+    return beers.map(({ id, description, name, abv, ibu, ebc, image_url: img, tagline, first_brewed: brewed, food_pairing: food }) => (
```
Comment (mi-chal) [inline src/services/beers.js:20]: Break down all properties into separate lines to improve readability

**Review by mi-chal — APPROVED** [review]

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx:47]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx:108]: Done!

File: src/services/beers.js (hunk 3 above)
Comment (dwantula) [reply src/services/beers.js:20]: Done!

**Pull request merged**